packages/landing/src/attribution.ts

```diff
@@ -1,24 +1,24 @@
 import { AttributionCollection, AttributionStac } from '@basemaps/shared/build/attribution';
-import { StacLicense } from '@basemaps/shared/build/stac';
-import { BBox, Wgs84 } from '@linzjs/geojson';
-import { View } from 'ol';
+import { BBox } from '@linzjs/geojson';
+import { Wgs84 } from '@linzjs/geojson/build/wgs84';
 import { Extent } from 'ol/extent';
 import OlGeometryLayout from 'ol/geom/GeometryLayout';
 import OlPolygon from 'ol/geom/Polygon';
-import OlMap from 'ol/Map';
-import MapEventType from 'ol/MapEventType';
 import { transformExtent } from 'ol/proj';
-import Source from 'ol/source/Source';
-import { MapOptions, MapOptionType, WindowUrl } from './url';
 
-const Copyright = `© ${StacLicense} LINZ`;
+function escape(text: string): string {
+    return text.replace(/</g, '&lt;');
```

Comment:
this doesnt seem the safest way to escape strings,  most people also do > and &

is there a better way of doing this?

Reply:
We can use the browser escape but it is deprecated. I think it would be the better option though. Otherwise I can at least escape `&` not much point in escaping `>`.

Reply:
I decided to escape `<` and `&`

Reply:
I am now escaping via the DOM